README.md

```diff
@@ -1,9 +1,14 @@
 # WhoWeAre
-Documents about Innovationsverbund Öffentliche Gesundheit (InÖG)
+Documents about Innovationsverbund Öffentliche Gesundheit (InÖG) (loosely translated: Public healt innovation network).
 
 ## What one can find in here
 
-Here we put documents about the Innovationsverbund Öffentliche Gesundheit. There is a lot of German involved, as we are an association based in Germany. We are currently working on English versions. Our statutes will always be in German due to legal reasons.
+Here we put documents about and projects the Innovationsverbund Öffentliche Gesundheit (InÖG).
```

Comment:
Here we put documents about the Innovationsverbund Öffentliche Gesundheit (InÖG) and share our projects?